README.md

```diff
@@ -19,15 +20,51 @@
 
 ---
 
-Wn is a Python library for exploring information in wordnets. Install
-it from PyPI and download some data:
+Wn is a Python library for exploring information in wordnets. 
+
+## Installation
+
+Install it from PyPI using **pip** and download some data:
 
 ```console
 $ pip install wn
 $ python -m wn download oewn:2022  # the Open English WordNet 2022
 ```
 
-Then start exploring:
+Or, install using **conda**, from conda-forge channel:
+
+```sh
+# Method-1
+conda install -c conda-forge wn
+
+# Method-2
+# Using: conda install CHANNELNAME::PACKAGENAME
+conda install conda-forge::wn
+```
```

Comment:
Let's just pick one method. Also, I think we should move the `download` step until after wn is installed by either method.

Reply:
To clarify, something like this:

> Install it from PyPI with **pip**:
> ...
> Alternatively, install it from conda-forge using **conda**:
> ...
> Once you've installed the software, download some data:
> ...

Also, I forgot to mention, try to keep the command styling the same. In the first example, it uses a `console` code block starting with the `$` prompt. Either use that or switch it to an `sh` block without `$` to match the second.

Reply:
I agree with you on the fact that all code blocks should follow the same style. Since, GitHub allows one to copy the code snippets from a code block by clicking on the top right corner of the code block, I would suggest we drop the `$` and use the more concise `sh` like you suggested. 

This will allow users to click-copy-paste the commands from the README file.

Reply:
Looks great, thanks